common/util/PropertiesUtil.kt

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Utility methods for interacting with `java.util.Properties` objects - a convenient representation of small amounts of config, which might typically be supplied to a Kotlin build script.